### ISSUE TITLE
"Message participant" link from consent manager and individual response views

### ISSUE THREAD

**Issue by kimberscott** [lookit/lookit-api#319]
**TL;DR**: A way to message the appropriate participant with a question about a consent video or response, without copying and pasting an ID.

**Narrative**
"As an Experimenter, I would like to be able to send an email to a participant to confirm consent when I see a consent video with a technical problem or no reading aloud. I would like to be able to send an email to a participant in response to a session I am viewing, either to remind him/her about another session or to clarify some information about this session. I would like to just click a button to transfer the appropriate ID to a draft email in the email view, instead of copying and pasting an ID, because I am never sure which ID is the right one to copy."

**Acceptance Criteria**

- [ ] From the consent manager view, a researcher can send an email to the participant associated with the currently selected session. This is true for accepted, pending, and rejected sessions. The participant's email selections are respected (i.e. this does not bypass setting an email type).
- [ ] From the individual responses view, a researcher can send an email to the participant associated with the currently selected session. The participant's email selections are respected.
- [ ] Have a current beta tester try it on staging.

**Implementation Notes**

**Comment by becky-gilbert** [comment]
This part of this issue
> From the individual responses view, a researcher can send an email to the participant associated with the currently selected session. The participant's email selections are respected.

Was completed in https://github.com/lookit/lookit-api/pull/1534.

I will leave this issue open until we handle the part about sending messages from the Consent Manager view:
> From the consent manager view, a researcher can send an email to the participant associated with the currently selected session. This is true for accepted, pending, and rejected sessions. The participant's email selections are respected (i.e. this does not bypass setting an email type).
